ts/features/bonus/cgn/store/reducers/merchants.ts

```diff
@@ -0,0 +1,43 @@
+import * as pot from "italia-ts-commons/lib/pot";
+import { getType } from "typesafe-actions";
+import { NetworkError } from "../../../../../utils/errors";
+import { Action } from "../../../../../store/actions/types";
+import { GlobalState } from "../../../../../store/reducers/types";
+import { cgnMerchants } from "../actions/merchants";
+
+export type CgnMerchantsState = {
+  list: pot.Pot<ReadonlyArray<any>, NetworkError>;
```

Comment:
```suggestion
  merchants: pot.Pot<ReadonlyArray<any>, NetworkError>;
```
Could it be a more descriptive name?
tip: if this store section includes only merchants, it can be directly the merchants list
`export type CgnMerchantsState = pot.Pot<ReadonlyArray<any>, NetworkError>;`

Reply:
@Undermaken 
I used merchants as the attribute name in the `CgnState` since this section should have inside the merchants list and the selected merchant to show it's detail avoiding a navigation parameter. This was how i figured out the store:
```typescript
{
  ...globalState,
  cgn: {
    ..cgnState,
    merchants: {
      list: [],
      selectedMerchant: {}
    }
  }
}
```

Reply:
Good for me if this section will store also other data